description: build fix from upstream

commit	312719ee1077e62c260f201d7b933e6e14fdff49
author	Stephan Hartmann <stha09@googlemail.com>
committer	Chromium LUCI CQ <chromium-scoped@luci-project-accounts.iam.gserviceaccount.com>
tree	5e56c59867ce7386ba6eb2824358d85c09f38801
parent	8b7432fbc8a98b8bc576145fbaea74d4c2458046 [diff] [blame]

IWYU: add cmath for roundf in prediction_common

Bug: 957519
Change-Id: Iad4ff22126aaf6aa31e1bc1701796773d50574d8
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3368091
Reviewed-by: Illia Klimov <elklm@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#958055}


--- a/components/permissions/prediction_service/prediction_common.cc
+++ b/components/permissions/prediction_service/prediction_common.cc
@@ -4,6 +4,8 @@
 
 #include "components/permissions/prediction_service/prediction_common.h"
 
+#include <cmath>
+
 #include "base/json/json_reader.h"
 #include "base/json/json_writer.h"
 #include "base/notreached.h"
